### PR TITLE
feat: Websocket communication between k6 Studio and browser

### PR DESCRIPTION
**Pull request by allansson**
## Description

This PR sets up WebSocket communication between k6 Studio and the recorded browser. It currently only sends a dummy event to the server and is logged in the renderer process.

## How to Test

1. Start a recording.
2. Open up DevTools in k6 Studio.
3. You should see a message being logged every ~5 seconds.

## Checklist

- [X] I have performed a self-review of my code.
- [ ] I have added tests for my changes.
- [X] I have run linter locally (`npm run lint`) and all checks pass.
- [X] I have run tests locally (`npm test`) and all tests pass.
- [X] I have commented on my code, particularly in hard-to-understand areas.

## Related PR(s)/Issue(s)

Closes https://github.com/grafana/k6-cloud/issues/3087

### CODE REVIEW

File: vite.main.config.ts [hunk 1]
```diff
@@ -23,7 +24,7 @@ export default defineConfig((env) => {
         external,
       },
     },
-    plugins: [pluginHotRestart('restart')],
+    plugins: [pluginHotRestart('restart'), tsconfigPaths()],
```
Comment (allansson) [inline vite.main.config.ts:27]: Allows the main process to use `@/` import paths.

File: vite.preload.config.ts [hunk 2]
```diff
@@ -22,7 +23,7 @@ export default defineConfig((env) => {
         },
       },
     },
-    plugins: [pluginHotRestart('reload')],
+    plugins: [pluginHotRestart('reload'), tsconfigPaths()],
```
Comment (allansson) [inline vite.preload.config.ts:26]: Allows the main process to use `@/` import paths.

File: src/schemas/recording/v1/browser.ts
Comment (allansson) [inline src/schemas/recording/v1/browser.ts:1]: I assumed we're going to want to write zod schemas for recordings, i.e. HAR files, at some point so I structured this like we've done with generator/settings schemas.

**Review by going-confetti — APPROVED** [review]
LGTM!

**Pull request merged**